vocabularies/Common.xml

```diff
@@ -695,6 +698,9 @@ The minimum structure is
         <Property Name="CollectionPath" Type="Edm.String" Nullable="false">
           <Annotation Term="Core.Description" String="Resource path of an OData collection with possible values, relative to the document containing the value list mapping" />
         </Property>
+        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" Nullable="true" DefaultValue="true"> 
```

Comment:
```suggestion
        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" Nullable="true"> 
```

Reply:
Keep default value and use `false`.

Reply:
If we use `DefaultValue="false"`, we don't need nullable.